src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Dialog/BitDialog.scss

```
@@ -9,6 +9,11 @@
     z-index: $zindex-modal;
     font-size: $tg-fs-sm;
     font-family: $tg-font-family;
+    // The root spans the whole screen but is only a frame for the two layers that are actually there:
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)